pkg/statsd/statsd.go

```diff
@@ -153,6 +166,18 @@ func (s *Server) RunWithCustomSocket(ctx context.Context, sf SocketFactory) erro
 	stage = stgr.NextStage()
 	stage.StartWithContext(receiver.RunMetrics)
 	for r := 0; r < s.MaxReaders; r++ {
+		// Open socket
+		c, err := sf()
```

Comment:
What happens to `c` opened on line 141/154? Should probably drop it.

Reply:
Oh, oops - merge issue. Definitely should be dropped